js/data.js

```diff
@@ -1,10 +1,12 @@
 /* exported data */
-
+// eslint-disable-next-line no-unused-vars
 let entries = [];
+// eslint-disable-next-line no-unused-vars
 let drafts = {
   drafts: [],
   renderedTitles: [],
-  nextDraftNum: 0
+  nextDraftNum: 0,
+  editing: null
 };
 // eslint-disable-next-line no-unused-vars
 const quotes = {
```

Comment:
It'd be clearer what these are being used for by adding them to the `exported` list, instead of disabling `no-unused-vars`.